webapp/src/modules/mapBuilder.js

```diff
@@ -222,143 +375,76 @@ export default class MapBuilder {
       .addTo(this._map);
   }
 
-  startRefreshTimer(interval) {
-    if (!this._refreshId && !this._ongoingTrip) {
-      this._refreshId = window.setInterval(() => {
-        this.getScooters(this._lastBounds);
-      }, interval);
+  _startRefreshTimer(interval) {
+    if (this._refreshId && this._ongoingTrip) {
+      return;
     }
+    this._refreshId = window.setInterval(() => {
+      this._getAndUpdateScooters(this._lastBounds, this._mcg);
+    }, interval);
   }
 
-  stopRefreshTimer() {
-    if (this._refreshId) {
-      clearInterval(this._refreshId);
-      this._refreshId = null;
+  _stopRefreshTimer() {
+    if (!this._refreshId) {
+      return this;
     }
+    clearInterval(this._refreshId);
+    this._refreshId = null;
     return this;
   }
 
-  // Remove markers in visible bounds to prevent duplicates
-  removeVisibleLayers(bounds) {
-    const removableMarkers = [];
-    this._mcg.eachLayer((marker) => {
-      if (bounds.contains(marker._latlng)) {
-        removableMarkers.push(marker);
-      }
+  _setLocateControl() {
```

Comment:
Added this here but [this ticket](https://github.com/lithictech/suma/pull/185) has all the correct changes (besides the "_" function prefix update)